docs/content/1.getting-started/3.quick-start.md

```diff
@@ -60,69 +60,148 @@ export default NuxtAuthHandler({
-
-Note: The backend can also be in the same Nuxt 3 application, e.g., have a look at this example in the `nuxt-auth` repository:
-- [full nuxt app](https://github.com/sidebase/nuxt-auth/tree/main/playground-local)
-    - its [backend](https://github.com/sidebase/nuxt-auth/tree/main/playground-local/server/api/auth)
-    - its [`nuxt.config.ts`](https://github.com/sidebase/nuxt-auth/blob/main/playground-local/nuxt.config.ts)
+You can customize each endpoint to fit your needs or disable it by setting it to `false`. For example you may want to disable the `getSession` endpoint.
+```ts
+{
+  auth: {
+    baseURL: '/api/auth',
+    provider: {
+      type: 'local',
+      endpoints: {
+        getSession: false
+      }
+    }
+  }
+}
+```
 
 ::alert{type="info"}
-The linked example-implementation only serves as a starting-point and is not considered to be secure.
+See [configuration](/configuration/nuxt-config) for all options.
 ::
 
-The backend must accept a request with a body like:
-```ts
-{
-    username: 'bernd@sidebase.io',
-    password: 'hunter2'
-}
+
+#### Sign In Example
+
+To perform a sign in, all you need to do is calling the `signIn` function from `useAuth` composable and pass the credentials expected by your backend.
+
+::alert{type="warning"}
+Note that `signIn` method will be successful only if a a `token` in the path specified `auth.provider.token` in the config is returned in the API response. Otherwise it will throw an error
+::
+
+Take this example where we're using the default options and require the user to enter his email, phone, and password to login. 
+
+```vue
+<script setup>
+  const email = ref('')
+  const phone = ref('')
+  const password = ref('')
+
+  const { signIn } = useAuth()
+  async function signInWithCredentials() {
+    // Probably you'll do some validation here before submitting to the backend
+    // ...
+
+    // This is the object that our backend expects for the `signIn` endpoint
+    const credentials = {
+      email: email.value
+      phone: phone.value
+      password: password.value
+    }
+
+    try {
+      // This sends a POST request to the `auth.provider.endpoints.signIn` endpoint with `credentials` as the body
+      await signIn(credentials)
+      alert('Successfully logged in!')
+    } catch (error) {
+      console.error(error)
+    }
+  }
+</script>
+
+<template>
+  <div>
+    <h1>Enter your credentials to continue</h1>
+    <input v-model="email" type="email" />
+    <input v-model="phone" type="tel" />
+    <input v-model="password" type="password" />
+    <button @click="signInWithCredentials()" />
+  </div>
+</template>
 ```
 
-and return a token that can be used to authenticate future requests in the response body, e.g., like:
+If the backend response included a `token` key, the user will be signed successfully.
+
 ```ts
+// Backend response
 {
-    token: {
-        accessToken: 'eyBlaBlub'
-    }
+  token: 'eyBlaBlub',
```

Comment:
In all of our examples (including the linked branch on the example) we return `token` as an object and not a string. 

https://github.com/sidebase/nuxt-auth/blob/main/playground-local/server/api/auth/login.post.ts#L30
https://github.com/sidebase/nuxt-auth-example/blob/add-local-provider/server/api/auth/login.post.ts#L30

I think it would make sense to mirror this in our docs, as this is how it appears in all examples!

Reply:
I agree. Will push a change